### ISSUE TITLE
BUG: Second org admin creating VC from org account gets errors

### ISSUE THREAD

**Issue by SimoneZaza** [alkem-io/client-web#7318]
**Describe the bug**
We have a bug for org admins creating expert VCs. When someone else from this org created a Space or VC first (and didn't add person B as an admin to this space), the second one trying to create a VC get's errors. This happens since they don't have the rights to create a subspace in the space, and can't add the vc. 

As we're moving away from using a space for the BOK, just a quick "fix" for now: When they start the flow, check if the user has the right privileges for the space, if not, inform them:

"It seems you don’t currently have the required permissions to add a Virtual Contributor to (Spacename). Please ensure you are an admin of this Space before proceeding."

**To Reproduce**

**Expected behavior**

As we're moving away from using a space for the BOK, just a quick "fix" for now: When they start the flow, check if the user has the right privileges for the space, if not, inform them:

"It seems you don’t currently have the required permissions to add a Virtual Contributor to (Spacename). Please ensure you are an admin of this Space before proceeding."

**Areas that will be affected**
_Optional, to be added by developer if relevant_

**Comment by bobbykolev** [comment]
TL;DR - the client already handles missing privileges.

With a new DB, Org, and users, the flow works as expected showing the following copy in case of insufficient privileges:
```
"title": "Insufficient Privileges",
"description": "You don't have the necessary privileges to complete the creation. Please contact the owners of this account."
``` 
I did hit an issue with the entitlements with an existing Org on the acceptance DB. 
`Entitlement space-flag-virtual-contributor-access is not available for License: 5fabfb18-5d73-424f-820a-589a3ac9b3a4`

I'd suggest either @valentinyanakiev or @techsmyth to double-check the entitlements logic related to existing Orgs.

Also, if we can get concrete information for the issue on prod (user, Org, error) it would be ideal. @SimoneZaza

**Comment by SimoneZaza** [comment]
Closing as we now have a completely new flow where space is no longer needed to create a knowledge base
